Doc/library/pathlib.rst

```diff
@@ -1052,6 +1052,20 @@ call fails (for example because the path doesn't exist).
       Added return value, return the new Path instance.
 
 
+.. method:: Path.absolute()
+
+   Make the path absolute, without resolving symlinks, and return a new path
```

Comment:
Should be `Path`  object?

Reply:
Not if we're aiming for consistency with the rest of the docs!